include/revng/ADT/STLExtras.h

```diff
@@ -449,10 +449,11 @@ static_assert(takeAsTupleExample() == 5);
 // Some views from the STL.
 // TODO: remove these after updating the libc++ version.
 //
-template<typename EnumType>
-[[nodiscard]] constexpr std::underlying_type<EnumType>::type // NOLINTNEXTLINE
-to_underlying(EnumType Value) {
-  return static_cast<std::underlying_type<EnumType>::type>(Value);
+template<typename EnumDefinition>
+[[nodiscard]] constexpr std::underlying_type<EnumDefinition>::type
+// NOLINTNEXTLINE
```

Comment:
Because the line is too long for it otherwise. I guess we could just add `to_underlying` to the linter exceptions, but this is going to go away the very next time we update `libc++`, so I though ugly but easily noticeable is the way to go.

Reply:
Ah, fuck. My bad. It got caught by the regex and I never noticed that it's not related to the model type system 🤦

Done.